tensorflow_blade/build.py

```diff
@@ -255,7 +260,7 @@ def package_whl_with_bazel(args):
 
 def test_with_bazel(args):
     with cwd(ROOT):
-        execute("bazel test //tests/...")
+        execute("bazel test --config=cuda //tests/...")
```

Comment:
Well, this worth a discussion. What we do internally is to generate all flags into `.bazelrc_gen`, to run a different configuration(maybe switching from CPU to GPU) one need to re-generate the `.bazelrc_gen` file. `bazel` commands are call without any `config`-like arguments. If we decide to use more config flags, we should generate as much as possible into `.bazelrc_gen`. 
Anyway, I've no preference.